README.md

```diff
@@ -161,6 +189,36 @@ You can control which version of Kodelet is installed:
 - **Development**: Use `latest` to get the newest features
 - **Testing**: Pin to specific versions to ensure reproducible results
 
+### Kodelet Configuration
+
+The action supports configurable Kodelet settings through YAML configuration content:
+
+```yaml
+# Use custom configuration content
+- uses: jingkaihe/kodelet-action@v0.1.4-alpha
+  with:
+    anthropic-api-key: ${{ secrets.ANTHROPIC_API_KEY }}
+    kodelet-config: |
```

Comment:
the `kodelet-config` in this doc is wrong, please use https://github.com/jingkaihe/kodelet/blob/main/config.sample.yaml as aa reference